modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -128,20 +129,20 @@ An example `unattend.xml` is provided, which maintains all the changes needed fo
          <TimeZone>Eastern Standard Time</TimeZone>
          <UserAccounts>
             <AdministratorPassword>
-               <Value>MyPassword</Value>
+               <Value>MyPassword</Value> <3>
                <PlainText>true</PlainText>
             </AdministratorPassword>
-            <LocalAccounts>
-               <LocalAccount wcm:action="add">
-                  <Description>Administrator</Description>
-                  <DisplayName>Administrator</DisplayName>
-                  <Group>Administrators</Group>
-                  <Name>Administrator</Name>
-               </LocalAccount>
-            </LocalAccounts>
          </UserAccounts>
       </component>
    </settings>
 </unattend>
 ----
+<1> Specify the `ComputerName`, which must follow the link:https://kubernetes.io/docs/concepts/overview/working-with-objects/names[Kubernetes' names specification]. These specifications also apply to Guest OS customization performed on the resulting template while creating new VMs.
+<2> Disable the automatic logon to avoid the security issue of leaving an open terminal with Administrator privileges at boot. This is the default value and should not be changed.
+<3> Replace the `MyPassword` placeholder with the password for the Administrator account. This prevents the built-in Administrator account from having a blank password by default. Follow the Microsoft's link:https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements[best practices for choosing a password].
 ====
+
+.. Once the `Sysprep` tool has completed, the Windows VM will power off. You must not use or power on this virtual machine anymore.
+
+. Convert the Windows VM to link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-846238E4-A1E3-4A28-B230-33BDD1D57454.html[a template in vCenter].
```

Comment:
We shouldn't target specific VMware-vSphere versions in the reference links;

In line [#70  ...clone an existing virtual machine](https://github.com/openshift/openshift-docs/pull/35728/files#diff-a9972807dbbd53e5423f681de126b7ec7a6e26d87bea6a6232b1e8d4e192a686R70) is also targeting version 7.0.